[DOCS] Adds disclaimer to cat APIs (#83382)
elastic/elasticsearch: docs/reference/cat/alias.asciidoc

<titleabbrev>cat aliases</titleabbrev>
++++

[IMPORTANT]
====
cat APIs are only intended for human consumption using the

> {kibana-ref}/console-kibana.html[Kibana console] or command line. They are _not_

Reviewer: Though this is taken from the existing https://www.elastic.co/guide/en/elasticsearch/reference/master/cat.html page, IMO (1) folks know what the Kibana console is without requiring a link and (2) the order of terms should be flipped, otherwise it sounds like there's a "Kibana command line":
```suggestion
command line or {kib} console. They are _not_
```

> intended for use by applications. For application consumption, we recommend

Reviewer: In general, I omit "we" unless absolutely necessary:
```suggestion
intended for use by applications. For application consumption, 
```

> using the <<aliases>> API.

Reviewer: ```suggestion
use the <<aliases>> API.
```
